server/src/controllers/userProfile.controller.js

```diff
@@ -0,0 +1,36 @@
+const uuid = require('uuid');
+const userSchema = require('../models/User.js');
+const appResponse = require('../utils/response');
+const DatabaseConnection = require('../db/database.helper');
+const Users = new DatabaseConnection('User');
+
+exports.userCreate = async (req, res) => {
+  const { body } = req;
+  body.id = uuid.v4();
+  console.log(body);
+
+  const user = await userSchema.validateAsync(body);
+  const response = await Users.create(user);
+
+  res.status(200).send(appResponse(null, response, true));
+}
+
+exports.getAllUsers = async (req, res) => {
```

Comment:
Refactor all the methods/functions in this class to be like the ones in other controllers (for consistency)